test/integration/order_integration_test.rb

```diff
@@ -11,6 +11,7 @@ def setup
                         last_name: 'Doe',
                         email: 'example@example.com')
     @order = Order.create(total_cost: 100, user_id: user.id)
+    @item = order.items.create(title: 'Pour Over')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.